trnsysGUI/pythonInterface/regimeExporter/renderDiagramOnPDFfromPython.py

```diff
@@ -101,9 +108,10 @@ def _simulateAndVisualizeMassFlows(
             )
             massFlowSolverVisualizer.slider.setValue(timeStep)
 
-            self._makeDiagramFiles(regimeName=regimeName)
+            self._makeNoFlowDiagramFiles(regimeName=regimeName)
```

Comment:
This should call makeDiagramFiles.

Reply:
Sorry missed that, we use the same fuction to make ones with flow. 
In that case, I think the previous name is fine